spec/support/shared_context.rb

```diff
@@ -0,0 +1,34 @@
+require 'json'
+
+RSpec.configure do |rspec|
+  # This config option will be enabled by default on RSpec 4,
+  # but for reasons of backwards compatibility, you have to
+  # set it on RSpec 3.
+  #
+  # It causes the host group and examples to inherit metadata
+  # from the shared context.
+  rspec.shared_context_metadata_behavior = :apply_to_host_groups
+end
```

Comment:
why do you add this config 👀

Reply:
of so many push I did and mistakes I thought those lines were necessary